assignments/callbacks.js

```diff
@@ -63,11 +63,12 @@ multiplyNums(10, 5, mult => {
 });
 
 function contains(item, list, cb) {
-  cb(item, list);
+  for (let index of items)
```

Comment:
for (let i = 0; i < list.length; i++){
    if (item === list[i]){
      answer = true;
      break
    }
    else {
      answer = false;
    }
  }
  return cb(answer)